seqr/views/utils/project_context_utils.py

```diff
@@ -118,63 +118,69 @@ def add_child_ids(response):
 
 
 def families_discovery_tags(families, genome_version, project=None):
-    families_by_guid = {f['familyGuid']: dict(discoveryTags=[], **f) for f in families}
+    families_by_guid = {f['familyGuid']: dict(discoveryGeneIds=[], **f) for f in families}
```

Comment:
we were previously returning a list of tags with transcripts and main and selected transcripts IDs, but then all the UI was doing was determining which of those is the single main gene to show. Moving that server side makes this much easier and removes the need to query clickhouse for most variants